wgsl/index.bs

```diff
@@ -1309,57 +1309,142 @@ repeatedly replacing a subsequence with its homoglyph.)
 
 A <dfn>context-dependent name</dfn> is a [=token=] used to name a concept, but only in specific grammatical contexts.
 The spelling of the token may be the same as an [=identifier=], but the token does not [=resolve=] to a declared object.
+This section lists the tokens used as [=context-dependent names=].
+
+### Attribute Names ### {#attribute-names}
 
-Section [[#context-dependent-name-tokens]] lists all such tokens.
+The [=syntax/attribute=] names are:
 
-## Built-in Value Names ## {#builtin-value-names}
+* <a for=attribute lt=align>`'align'`</a>
+* <a for=attribute lt=binding>`'binding'`</a>
+* <a for=attribute lt=builtin>`'builtin'`</a>
+* <a for=attribute lt=compute>`'compute'`</a>
+* <a for=attribute lt=const>`'const'`</a>
+* <a for=attribute lt=fragment>`'fragment'`</a>
+* <a for=attribute lt=group>`'group'`</a>
+* <a for=attribute lt=id>`'id'`</a>
+* <a for=attribute lt=interpolate>`'interpolate'`</a>
+* <a for=attribute lt=invariant>`'invariant'`</a>
+* <a for=attribute lt=location>`'location'`</a>
+* <a for=attribute lt=size>`'size'`</a>
+* <a for=attribute lt=vertex>`'vertex'`</a>
+* <a for=attribute lt=workgroup_size>`'workgroup_size'`</a>
+
+### Built-In Value Names ### {#builtin-value-names}
```

Comment:
To match the section they are defined in.

```suggestion
### Built-in Value Names ### {#builtin-value-names}
```

Reply:
Fixed!